src/main/java/io/confluent/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -289,12 +270,11 @@ private boolean isIntegralPrimitiveType(Object incrementingColumnValue) {
 
   @Override
   public String toString() {
-    return "TimestampIncrementingTableQuerier{"
-           + "name='" + name + '\''
-           + ", query='" + query + '\''
-           + ", topicPrefix='" + topicPrefix + '\''
-           + ", timestampColumn='" + timestampColumn + '\''
-           + ", incrementingColumn='" + incrementingColumn + '\''
-           + '}';
+    return "TimestampIncrementingTableQuerier{" +
+        "incrementingColumn='" + incrementingColumn + '\'' +
```

Comment:
I was expecting checkstyle to complain about `+` not being at the start of the line. Checking at the jenkins build I see it does indeed (along with a few more errors).